guides/common/modules/con_upgrade-paths.adoc

```diff
@@ -14,8 +14,10 @@ The high-level steps in upgrading {Project} to {ProjectVersion} are as follows:
 
 . Upgrade {ProjectServer} to {ProjectVersion}.
 For more information, see xref:Project_Upgrade_Considerations_{context}[].
+ifndef::foreman-deb[]
```

Comment:
alternative is to flip line 17 and 18.

Reply:
For Debian, this only shows one item in the list. Which is not great and flipping 17 and 18 could help. Have considered, as an alternative, simply dropping the "High-level upgrade steps" list for all builds? The section is named "Upgrade paths", so I would say that an overview of the upgrade does not belong there anyway. Additionally, from looking at the table of contents and considering the overall flow of the document, I think users should be able to follow the upgrade steps just fine without this summary.

Reply:
I have flipped 17 and 18. I think the concept would be almost empty without those two steps. I am OK with the following solutions for my PR:

* keep as is
* drop 11-end
* drop the whole file because without line 11-end, the file does not really show any "upgrade paths" as the title suggests. I would then move line 7+8 for all build targets to another file because we should not really recommend users to skip versions during upgrades.

What do you think is best?

Reply:
I have a slight preference towards dropping the file, but  that option would also require the most effort from you. Considering that the goal is to make progress with the migration from the Manual, "keep as is" will work just fine too :)